snapshot-labs/sx-monorepo
# feat: add link to label

### Summary



Depends on https://github.com/snapshot-labs/sx-monorepo/pull/910

This PR enable label to link to the proposal page, filtered by the given label

### How to test

1. Go to a space proposals page with labels
2. Click on label in the proposals title
3. It should redirect to the proposals page, with the given label filter
4. Repeat same steps, for label inside the single proposal page
